src/background/index.ts

```diff
@@ -113,11 +113,14 @@ chrome.runtime.onMessage.addListener((request, _sender, sendResponse) => {
 // TODO It's better to move this to the provider class. Let's figure out how to do it later.
 // Removes cookies for captcha.website to enable getting more tokens in the future.
 chrome.cookies.onChanged.addListener((changeInfo) => {
-    if (
-        !changeInfo.removed &&
-        changeInfo.cookie.domain === '.captcha.website' &&
-        changeInfo.cookie.name === 'cf_clearance'
-    ) {
-        chrome.cookies.remove({ url: 'https://captcha.website', name: 'cf_clearance' });
+    const cloudflareDomains = ['captcha.website', 'issuance.privacypass.cloudflare.com']
+    for (const domain of cloudflareDomains) {
+        if (
+            !changeInfo.removed &&
+            changeInfo.cookie.domain === `.${domain}` &&
```

Comment:
What about removing the outer loop:
```suggestion
            cloudflareDomains.includes(domain) &&
```
where `cloudflareDomains` is modified to have a period before?